addons/bus/static/src/services/bus_service.js

```diff
@@ -27,11 +27,11 @@ export const busService = {
         const notificationBus = new EventBus();
         const subscribeFnToWrapper = new Map();
         let worker;
-        let isActive = false;
         let isInitialized = false;
         let isUsingSharedWorker = browser.SharedWorker && !isIosApp();
         const startedAt = luxon.DateTime.now().set({ milliseconds: 0 });
         const connectionInitializedDeferred = new Deferred();
+        const state = reactive({ isActive: false });
```

Comment:
Is that actually working? The getter returns the value not the reactive object.
Is the full chain of this.env.service.bus_service actually reactive for this to work?
That's possible but I'm a little bit surprised.

Reply:
It works. The getter reads the reactive, and the service writes on the reactive. I'd say its a classic flow?

Reply:
This is what components make it look like, but they hide a lot of the details.

It's much more complex, there needs to be a full chain of reactive of which at least one has an observable callback for anything to happen.

Reading a value from a reactive only triggers the observable of that particular reactive, which in this case is non-existent.

It's only fine because somehow the full chain is wrapped starting from the compute reactive, and if any part of the chain enforced markRaw for example it would not work.